src/index.js

```diff
@@ -1,11 +1,41 @@
-export default sheet => {
-  const lessVars = {};
-  const matches = sheet.match(/@(.*:[^;]*)/g) || [];
+import stripComments from 'strip-json-comments';
+
+const varRgx = /^[@$]/;
+const followVar = (value, lessVars, defaults) => {
```

Comment:
`defaults` doesnt feel like the right name for this. But it's a tricky one. Im thinking something like `dictionary`? Did you have any other ideas?